config/brakeman.ignore

```diff
@@ -0,0 +1,25 @@
+{
+  "ignored_warnings": [
+    {
+      "warning_type": "Format Validation",
+      "warning_code": 30,
+      "fingerprint": "5627a6e09b94a01ea882ed3875d48192b41518fcbfc6f8471bb11a787a870632",
+      "check_name": "ValidationRegex",
+      "message": "Insufficient validation for `emails` using `/([^,]+)/`. Use `\\A` and `\\z` as anchors",
```

Comment:
why was this ignored?

Reply:
Because I have to check that the string are a list of emails separated with `,` (comma). If I add `\\A` and `\\z` as anchors it takes the whole string as a single one and does not match anything. WDYT?